content/en/docs/reference/command-line-tools-reference/feature-gates/security-context-deny.md

```diff
@@ -9,5 +9,7 @@ stages:
   - stage: alpha 
     defaultValue: false
     fromVersion: "1.27"
+    fromVersion: "1.29"
```

Comment:
Just a small adjustment: instead of "fromVersion", it should be "toVersion". 
Other than this, everything else in the PR looks good.

```suggestion
    toVersion: "1.29"
```

Reply:
Fixed.